rules/windows/lateral_movement_cmd_service.toml

```diff
@@ -22,15 +22,14 @@ tags = ["Elastic", "Windows"]
 type = "eql"
 
 query = '''
-/* dependent on a wildcard for remote path */
-/* replace winlog.event_data.OriginalFileName with process.pe.original_file_name once field is available in winlogbeat */
-
 sequence by process.entity_id with maxspan=1m
   [process where event.type in ("start", "process_started") and
-     (process.name == "sc.exe" or process.pe.original_file_name == "sc.exe" or
-        winlog.event_data.OriginalFileName == "sc.exe") and
-     wildcard(process.args, "\\\\*") and wildcard(process.args, "binPath*", "binpath*") and
-     process.args in ("create", "config", "failure", "start")]
+     (process.name == "sc.exe" or process.pe.original_file_name == "sc.exe") and
+                                                   /* case insensitive */
+      process.command_line:"* \\\\* binPath=*" and process.args:"create" or
+                                                   process.args:"config" or
+                                                   process.args:"failure" or
+                                                   process.args:"start"]
```

Comment:
```suggestion
      wildcard(process.args, "\\\\*") and wildcard(process.args, "binPath=*") and (
          process.args : "create" or
          process.args : "config" or
          process.args : "failure" or
          process.args : "start")]
```

Reply:
This will `and` the same multi-value field. Does using a method bypass the optimization bug?

Reply:
as long as we don't AND with `==` specifically on a field, we should be okay. so an AND on `:` is still safe, or functions

Reply:
thanks for the distinction